src/main/webapp/enroll.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <title>Enrollment Page</title>
+      <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    
+    <div class="jumbotron">
+        <h1>Product Release</h1>      
+    </div> 
+
+    <div class="parent-container d-flex">
+        <div class="container">
+            <div class="row">
+                <div class="col">
+                    <div class="container mt-3">
+                        <h2>SDKs</h2>
+                        <form action="/action_page.php">
+                            <select name="sdks" class="custom-select mb-3">
+                            <option selected>Select SDK</option>
+                            <option value="firebase_common">Firebase Common</option>
+                            <option value="firebase_messaging">Firebase Messaging</option>
+                            <option value="firebase_idd">Firebase Idd</option>
+                            <option value="firebase_storage_api">Firebase Storage API</option>
+                            <option value="firebase_ads">Firebase Ads</option>
+                            <option value="tagmanager">Tagmanager</option>
+                            </select>
+                            <button type="submit" class="btn btn-primary">Submit</button>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="container">
+            <div class="row">
+                <div class="col">
+                    <div class="card text-white">
+                        <div class="card">
+                            <h2 class="card-title">M78</h2>
+                        </div>
+                    </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <hr>
+   
+   <div class="border">
+        <div class="container" >
+        <h2>SDK Library Detail</h2>
+        <form action="/action_page.php" class="text-center">
+            <div class="form-group">
+            <label for="crv">Current Version</label>
+            <input type="text" class="form-control" id="crv" name="currentversion">
+            </div>
+
+            <hr>
+            <div class="form-group">
+            <label for="nv">Current Version</label>
```

Comment:
The previous item is also called 'Current Version'. Did you want to add something else here?